shopify/wishlist-template/api/actions/sendInStockEmails.ts

```diff
@@ -0,0 +1,80 @@
+import { renderEmail } from "../utilities";
+
+export const run: ActionRun = async ({
+  params,
+  logger,
+  api,
+  connections,
+  emails,
+}) => {
+  const { customers, variant, productId, shopId } = params;
+
+  if (!variant) {
+    throw new Error("Variant is undefined");
+  }
+
+  const { id, title } = variant;
+
+  const [shop, product] = await Promise.all([
+    api.shopifyShop.findOne(shopId as string, {
+      select: {
+        name: true,
+        domain: true,
+      },
+    }),
+    api.shopifyProduct.findOne(productId as string, {
+      select: {
+        handle: true,
+        title: true,
+      },
+    }),
+  ]);
+
+  if (customers) {
+    for (const customer of customers) {
+      await emails.sendMail({
```

Comment:
Why not `Promise.all` here?
I feel like we would get more benefit due to the variable number of `customers`

Reply:
Fixed this with p-map